node/lib/packaging/PackageInfo.ts

```diff
@@ -0,0 +1,173 @@
+
+/**
+ * Information about an OpenT2T package, as loaded from the package.json.
+ */
+export class PackageInfo {
```

Comment:
This is not meant to cover everything that might be in a package.json -- it's just the stuff that we decide is useful for OpenT2T to parse out of it. NPM package.json already defines an "author" property, so we could easily parse that into this class if we need to, but I don't know of a reason right now.

---
In reply to: [73275025](https://github.com/openT2T/opent2t/pull/2#discussion_r73275025) [](ancestors = 73275025)